lib/maze/servlets/servlet.rb

```diff
@@ -19,6 +22,10 @@ def initialize(server, request_type, schema=nil)
         @request_type = request_type
         @requests = Server.list_for request_type
         @schema = JSONSchemer.schema(schema) unless schema.nil?
+
+        if Maze.config.repeater_api_key && REPEATED_REQUEST_TYPES.include?(@request_type)
+          @repeater = Maze::RequestRepeater.new(@request_type)
+        end
```

Comment:
I don't like that this is nilable and the logic for the request type is duplicated (here & `url_for_request_type)

I think it'd be better for `RequestRepeater` to handle whether it is enabled so the `Servlet` can just call it unconditionally